server/network/webhooks.py

```diff
@@ -89,6 +89,19 @@ def modcall(self, char, ipid, area, reason=None):
             description=description,
         )
 
+    def login(self, client=None, loginprofile=""):
```

Comment:
plz check if "login_webhook" is in self.server.config, otherwise it will error and not allow you to launch the server if the new config is not present